### ISSUE TITLE
CONAN_UPLOAD_RETRY doesn't affect anything

### ISSUE THREAD

**Issue by Artalus** [conan-io/conan-package-tools#244]
In `packager.py ConanMultiPackager` there is this line
```self.upload_retry = upload_retry or os.getenv("CONAN_UPLOAD_RETRY", 3)```
but `self.upload_retry` doesn't get passed anywhere later.

**Comment by lasote** [comment]
You are right. Some versions ago I changed the invocation to conan by command line with calling the python API. I will take a look
